recipes/imagepy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "imagepy" %}
+{% set version = "0.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "511556789365fffa4827f771ca246d10bbb7fd7a07240d6a29c39b3e44e83e1e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-image >=0.12
+    - shapely >=1.0
+    - wxpython >=4.0
+    - numba >=0.29
+
+test:
+  imports:
+    - imagepy
+
+about:
+  home: https://github.com/Image-Py/imagepy
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: ''
```

Comment:
Added PR ( https://github.com/Image-Py/imagepy/pull/16 ) upstream, which should ensure the license file can be included in packages in the future.

Reply:
@jakirkham need I re upload to pypi first?

Reply:
Yes, so now that you've merged that PR the next upload to PyPI will have the license file in it. If you want to do a patch release now that includes that, then you could update the recipe to use that version and set `license_file: LICENSE`. Otherwise you could just remember to do it for the next release.

Reply:
Or could copy the license file over here for now, in which case it can be dropped from the recipe on the next update. Anyways lots of options to choose from here, feel free to pick what works best for you.